schemas/mobility.schema

```diff
@@ -0,0 +1,25 @@
+SET client_min_messages TO WARNING;
+BEGIN;
+
+DROP SCHEMA IF EXISTS $PG_SCHEMA CASCADE;
+CREATE SCHEMA $PG_SCHEMA_MOBILITY;
+
+CREATE TABLE $PG_SCHEMA_MOBILITY.mobility (
+locale_id  					   INTEGER NOT NULL,
+timestamp  					   TIMESTAMP NOT NULL,
+population_staying_at_home     INTEGER,
```

Comment:
Trying to shorten this as well. Any preference between (or general dislike towards) the following two?

1. `pop_staying_home` + `pop_not_staying_home`
2. `pop_home` + `pop_mobile`

Reply:
same as comment above.

Reply:
We can toss a coin on this one then.

Reply:
Picked #2. Done.